spyderlib/spyder.py

```diff
@@ -580,6 +580,14 @@ def setup(self):
                                             context=Qt.WidgetShortcut)
             self.register_shortcut(self.replace_action, "Editor",
                                    "Replace text")
+            self.file_switcher_action = create_action(self, _('File switcher...'),
+                                            icon='filelist.png',
+                                            tip=_('Fast switch between files'),
+                                            triggered=self.call_file_switcher,
+                                            context=Qt.ApplicationShortcut)
+            self.register_shortcut(self.file_switcher_action, "_",
+                                           "file switcher")

```

Comment:
Align this line to the previous left parenthesis, like this

```python
self.register_shortcut(self.file_switcher_action, "_",
                       "File switcher")
```

and capitalize `File switcher`